src/app/stores/Store.js

```diff
@@ -107,6 +108,10 @@ class Store {
   updateSubjectHeading(data) {
     this.setState({ subjectHeading: data });
   }
+
+  updateDrbbResults(data) {
+    this.setState({ drbbResults: data });
+  }
 }
```

Comment:
Should `updateDrbbResults` also appear in the call to `bindListeners`?

Reply:
Yes.

Reply:
Thanks for pointing that out